Bottom: 5bde029ddde9a74046932db59bb5d4767023ca2a
Top:    e06cef9d1212c7ffa54b9f6396c4bcd36791b090
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-18 20:43:45 +0000

polygon.h: Reduce circ_segs to 8 for debug clarity


---

diff --git a/src/polygon.h b/src/polygon.h
index 369efea..0643765 100644
--- a/src/polygon.h
+++ b/src/polygon.h
@@ -34,7 +34,7 @@
 
 /* Implementation constants */
 
-#define POLY_CIRC_SEGS 40
+#define POLY_CIRC_SEGS 8 //40
 #define POLY_CIRC_SEGS_F ((float)POLY_CIRC_SEGS)
 
 #if 0
